src/material/snack-bar/snack-bar-ref.ts

```diff
@@ -128,4 +128,9 @@ export class MatSnackBarRef<T> {
   onAction(): Observable<void> {
     return this._onAction;
   }
+
+  /** Gets an observable that is notified when the snack bar is announced to screen readers. */
+  onAnnounce(): Observable<void> {
+    return this.containerInstance._onAnnounce;
+  }
```

Comment:
What's the rationale for making this a public API?

Reply:
On second thought, this doesn't need to be public and I don't see a great benefit in making it so. I've removed this from here and just kept it in the `_SnackBarContainer` interface. Moved the call to subscribe to `_onAnnounce` from `_animateSnackBar` to `_attach` so I can use the `_SnackBarContainer` reference rather than `MatSnackBarRef`